src/problem-analysis/state-of-the-art.tex

```diff
@@ -0,0 +1,85 @@
+% textidote: ignore begin
+\section{State of the Art}\label{sec:state-of-the-art}
+% textidote: ignore end
+
+The following section will look at and analyze existing solutions and their approach to visualizing statistics and
+analytics, while looking for inspirations or areas to improve upon.
+The chosen solutions are \acrlong{ga4} and Microsoft BI, as they are both widely used for commercial purposes and
+provide a wide range of features.
+% TODO change microsoft bi to a different analytics tool that could fit better for the solution
```

Comment:
@Danar435 Not a code review comment, but content comment for the future: Power BI is the shit on the market atm. Why would you want to write about something else?